pom.xml

```diff
@@ -141,7 +141,7 @@
 
     <!-- These two versions are interdependent. -->
     <version.directory.maven.plugin>1.0</version.directory.maven.plugin>
-    <version.dspace.oai>4.2.0</version.dspace.oai>
+    <version.dspace.oai>5.2.0</version.dspace.oai>
```

Comment:
Name of the parameter change to accommodate the new library?

Reply:
done. renamed to `version.gdcc.xoai`